docs/modules/configuration/pages/dynamic-config-persistence.adoc

```diff
@@ -0,0 +1,208 @@
+= Configuring Dynamic Configuration
```

Comment:
minor: Since we change this back to old name, please feel free to rename it. Like `configuring-dynamic-configuration.adoc` for example.